JudoPay.tsx

```diff
@@ -58,14 +58,62 @@ export type { JudoPBBAConfiguration } from './types/JudoPBBATypes'
 export { JudoPBBAButton }
 
 class JudoPay {
-    public isSandboxed = true
+
+    //------------------------------------------------------------------
+    // Private properties
+    //------------------------------------------------------------------
 
     private readonly authorization: JudoAuthorization
 
+    //------------------------------------------------------------------
+    // Public properties
+    //------------------------------------------------------------------
+
+    /**
+     * This property is used to toggle the sandbox environment on and off.
+     * You can use the sandbox environment to test our SDK features.
+     */
+    public isSandboxed = true
+
+    //------------------------------------------------------------------
+    // Initializers
+    //------------------------------------------------------------------
+
+    /**
+     * The designated initializer that is used to configure the JudoPay session, based
+     * on a provided object that implements the JudoAuthorization interface.
+     *
+     * @param authorization - can be either a JudoBasicAuthorization (token & secret) or
+     * a JudoSessionAuthorization (token & payment session) instance.
+     */
     constructor(authorization: JudoAuthorization) {
         this.authorization = authorization
     }
 
+    //------------------------------------------------------------------
+    // SDK Features
+    //------------------------------------------------------------------
+
+    /**
+     * A method used to verify if any Pay By Bank App supported banking apps are installed
+     * on the device.
+     *
+     * This can be useful for conditionally rendering the PBBA button only if the bank app
+     * is available (PBBA won't work otherwise).
+     */
+    public isBankingAppAvailable(): boolean {
```

Comment:
Should this method also be async and return Promise<boolean> since that's the only way to communicate with native?

Reply:
D:
You're probably right

Reply:
Although this still works if you call the method like this:
`isAvailable = await judo.isBankingAppAvailable()`
so not sure.